### PR TITLE
Attach GitRev to marconi-chain-index nix-built binary 

### PR DESCRIPTION
**Pull request by zeme-wana**

### CODE REVIEW

File: nix/cabal-project.nix [hunk 1]
```diff
@@ -93,10 +93,21 @@ let
   };
 
 
+  overlays = [
+    (_: prev: {
+      hsPkgs = pkgs.pkgsBuildBuild.setGitRevForPaths
+        (inputs.self.rev or "0000000000000000000000000000000000000000") [
+        "marconi-chain-index.components.exes.marconi-chain-index"
```
Comment (zeme-wana) [inline nix/cabal-project.nix:100]: You can add more executables here if you wish

Comment (zeme-wana) [reply nix/cabal-project.nix:100]: Also note that this will only work (locally) if your git tree is clean.

**Comment by koslambrou** [comment]
@zeme-wana Was this PR part of #167 ?

**Comment by zeme-wana** [comment]
> @zeme-wana Was this PR part of #167 ?

Yes, I'll close. Thanks

**Pull request closed without merging**